app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,54 @@
+class CommentsController < ApplicationController
+  before_action :set_repository
+  respond_to :js, :html
+
+  # POST /repositories/1/comments
+  # POST /repositories/1/comments.json
+  def create
+    @comment = @repository.comments.new(comment_params)
+    @comment.author = current_user
+    authorize @comment
+
+    if @comment.save
+      @repository.create_activity :create_comment,
+                                  owner:      current_user,
+                                  parameters: {
```

Comment:
I don't really have a strong opinion here. Also, we haven't defined really which kinds of actions deserve an activity :)

So, even if @jloehel is going to kill me for saying this :D, let's do as @flavio says. This will simplify this PR a *lot* :)